src/main/java/mate/academy/Main.java

```diff
@@ -4,20 +4,19 @@
 import mate.academy.model.User;
 import mate.academy.service.AuthenticationService;
 import mate.academy.service.AuthenticationServiceImpl;
-import mate.academy.service.OrderService;
-import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
-            user = authenticationService.login("bob", "1234");
+            authenticationService.login("bob", "1234");
```

Comment:
The log message for a successful login attempt is missing. It is important to log both successful and failed attempts for tracking purposes. Consider adding a log message with the user's login for a successful authentication.